app/data/helpers/generators.js

```diff
@@ -0,0 +1,72 @@
+
+
+const faker = require('faker')
+faker.locale = 'en_GB'
+
+exports.firstName = (sex) => {
+  if(sex === 'male') {
+    return faker.helpers.randomize([
```

Comment:
You're missing:

- Bruce
- Wade
- Harold
- Peter
- James
- Taneleer
- Herman

at least...

Reply:
I need surnames for all the first names haha